hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestColumnStatsIndex.scala

```diff
@@ -411,6 +380,42 @@ class TestColumnStatsIndex extends HoodieClientTestBase {
     )
   }
 
+  private def validateColumnStatsIndex(testCase: ColumnStatsTestCase,
```

Comment:
This method is extracted from the existing logic of validating column stats index: (1) comparing the actual column stats index content with provided Json file, (2) for parquet files, construct the column stats index from the parquet file and compare that with column stats index from metadata table.